pkg/minishift/registration/redhat.go

```diff
@@ -51,17 +52,36 @@ func (registrator *RedHatRegistrator) CompatibleWithDistribution(osReleaseInfo *
 }
 
 func (registrator *RedHatRegistrator) Register(param *RegistrationParameters) error {
-	if output, err := registrator.SSHCommand("sudo -E subscription-manager version"); err != nil {
+	output, err := registrator.SSHCommand("sudo -E subscription-manager version")
+	if err != nil {
 		return err
-	} else {
-		if strings.Contains(output, "not registered") {
+	}
+	if strings.Contains(output, "not registered") {
+		for i := 1; i < 4; i++ {
+			if param.Username == "" {
+				param.Username = param.GetUsernameInteractive("Username to register to RHSM")
```

Comment:
https://access.redhat.com/articles/63269 that is official acronym. 

> Thats why I suggested Username to register to Red Hat. @praveenkumar any suggestions?

`Username to register to Red Hat` does look way to generic but I can go with something like `Username to register to RHSM/RHD` if that make sense?